xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -248,10 +305,10 @@
         <Parameter Name="typeName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="typeName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The resource name.</param>
+        <param name="value">The value of the resource in string form understood by the type's <see cref="T:System.ComponentModel.TypeConverter" />.</param>
+        <param name="typeName">An assembly qualified type name of the resource</param>
+        <summary>Adds a resource of the specified type represented by a string value. If the type is a primitive type, the value will be converted using <see cref="T:System.ComponentModel.TypeConverter" /> by the writer to that primitive type and stored in the resources in binary format. If the type is not a primitive type, the string value will be stored in the resources as a string and converted with a <see cref="T:System.ComponentModel.TypeConverter" /> for the type when reading the resource. This conversion is done to avoid activating arbitrary types during resource writing.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

If the type is a primitive type, the value will be converted using <xref:System.ComponentModel.TypeConverter> by the writer to that primitive type and stored in the resources in binary format. If the type is not a primitive type, the string value will be stored in the resources as a string and converted with a <xref:System.ComponentModel.TypeConverter> for the type when reading the resource. This conversion is done to avoid activating arbitrary types during resource writing.

         ]]></format>
         </remarks>
```